pkg/sql/delete_range.go

```diff
@@ -119,10 +123,18 @@ func maybeCreateDeleteFastNode(
 		return nil, false
 	}
 
+	i := 0
```

Comment:
[nit]: just use the iteration variable that the for loop provides (for i, tableEntry := range ... )

Reply:
Or just allocate `interleavedDesc` with len = 0, capacity = len(fkTables) and append.

Reply:
`fkTables` is a `map`, so i can't use a range iterator and get the index (like when iterating over a slice). Either way, your second proposal is best, I think.

Reply:
ack woops.